### DISCUSSION TITLE
Rejected my requests of GitHub Global Campus

### DISCUSSION THREAD

**Discussion by gitamaria** [community/community#56650] [GitHub Education]
### Hi everyone,

Your application cannot be reviewed until you fix the following:
You are unlikely to be verified until you have completed your [GitHub user profile](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile) with your full name as it appears in your academic affiliation document plus a short bio.
Your GitHub account was too recently created. Please apply again in a few days.
Please consider using your device camera to submit academic affiliation documents. Uploaded images are more easily manipulated and are therefore less trustworthy.

**Comment by jge162** [comment]
Says you need to wait. Your account is too new.
